tfutils/base.py

```diff
@@ -113,6 +113,7 @@ def __init__(self,
                  sess=None,
                  global_step=None,
                  cache_dir=None,
+                 ignore_same_loc = False,
```

Comment:
@chengxuz can you use the PEP8 style here?  e.g. no spaces between the argument and the default value? and below too?